scripts/checkMissingTranslations.ts

```diff
@@ -0,0 +1,59 @@
+import { Resource, ResourceKey } from 'i18next'
+import i18n from '../src/i18n'
+
+const checkRecursiveTranslation = (
+  searchingPath: string,
+  defaultLanguageObject: ResourceKey,
+  translatedLanguageObject: ResourceKey,
+) => {
+  if (typeof defaultLanguageObject === 'string' || typeof translatedLanguageObject === 'string') {
+    if (typeof defaultLanguageObject === 'object') {
+      console.log(
+        `📙 Found a string for path ${searchingPath} while is and object for the default language`,
```

Comment:
I don't understand what this log message is trying to convey. Could you please explain?

Reply:
This statement is the default case to make the recursive works and It will cover the following case
default translation
```json
{
  "translations":{
    "patient":{
      "errors":{
        "foo":"bar"
      }
    }
  }
}
```
comparing translation

```json
{
  "translations":{
    "patient":{
      "errors":"foo"
    }
  }
}
```
Now image to call checkRecursiveTranlation with
1) searchPath = translations->patient->errors
2) `{"foo":"bar"}`
3) `"foo"`

The comparing translation has an issue because it is directly a string instead of an object